index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link
+        href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
+        rel="stylesheet">
+    <title>Portfolio</title>
+</head>
+
+<body>
+    <main class="main-block">
+        <header class="head-menu">
+            <a class="logo" href="#">My Logo</a>
+            <nav>
+                <img class="ham-menu" src="./Images/ham-Icon.png" alt="menu">
+            </nav>
+        </header>
+        <section class="home-page">
+            <h1 class="intro">I’m Kartikay
+                Glad to see you!</h1>
+            <p class="intro-description">I’m a software developer! I can help you
+                build a product, feature or website. Look through some of my work and
+                experience! If you like what you see and
+                have a project you need coded, don’t hesitate to contact me.</p>
+            <h3 class="bottom-connect">LET’S CONNECT</h3>
+            <ul class="icon-container">
+                <li><a class="social-icon" href="#"><img src="./Images/Twitter-icon.png" alt="Twitter"></a></li>
```

Comment:
### OPTIONAL

- [ ]  Instead of using pictures for icons you can use [font-awesome-icons](https://fontawesome.com/start).